samples/custom-voice/csharp/CustomVoiceSample/Program.cs

```diff
@@ -0,0 +1,389 @@
+﻿
+
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Sas;
+using Microsoft.CognitiveServices.Speech;
+using Microsoft.CognitiveServices.Speech.Audio;
+
+public class Program
+{
+    private const string subscriptionKey = "YourSubscriptionKey";
+    private const string region = "YourServiceRegion. E.g., 'eastus'";
+
+    public static async Task ProfessionalVoiceTestAsync()
+    {
+        var client = new CustomVoiceClient(region, subscriptionKey);
+
+        var projectId = "professional-voice-project-1";
+        var consentId = "professional-voice-consent-1";
+        var trainingSetId = "professional-voice-training-set-1";
+        var modelId = "professional-voice-model-1";
+        var endpointId = Guid.NewGuid();
+
+        // Below are parameters for blob storage. You need to replace them with your own blob storage account information.
+        var blobAccountName = "YourBlobAccountName. E.g., 'voicetest'.";
+        var blobAccountKey = "YourBlobAccountKey";
+        var containerName = "YourContainerName.";
+        var consentblobPrefix = "YourBlobPrefix. E.g., consents";
+        var trainingDataBlobPrefix = "YourTrainingDataBlobPrefix. E.g., professionalvoice/trainingset";
+
+        // You can find sample script and audio file here.
+        // https://github.com/Azure-Samples/Cognitive-Speech-TTS/blob/master/CustomVoice/Sample%20Data/Individual%20utterances%20%2B%20matching%20script/SampleScript.txt
+        // https://github.com/Azure-Samples/Cognitive-Speech-TTS/blob/master/CustomVoice/Sample%20Data/Individual%20utterances%20%2B%20matching%20script/SampleAudios.zip
+        //Pleae unzip audio file. Put both audio and script file in foler below.
+        var localTrainingDataFolder = "YourLocalTrainingDataFolder. E.g., C:\\trainingset";
+
+        try
+        {
+            // Step 1: Create a project
+            var project = await client.CreateProjectAsync(projectId, ProjectKind.ProfessionalVoice, "Test project for professional voice");
+            Console.WriteLine($"Project created. project id: {project.Id}");
+
+            // Step 2: Create a consent
```

Comment:
In my python sample code, I wrapped each step as a function.
1. It makes user can reuse these functions in their product code.
2. Some step takes long time. It help user to comment off previous steps, and run following steps.